prow/jobs/kyma/tests/end-to-end/backup-restore-test/backup-restore-test.yaml

```diff
@@ -0,0 +1,56 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  skip_report: false
```

Comment:
have you checked that this job is passing? maybe it will be safer to set skip_report to true and check if it is pass and then if everything will be fine set skip_report again to true on another PR.